changelog.d/6451.bugfix

```diff
@@ -0,0 +1 @@
+Change the index on the `e2e_cross_signing_signatures` table to be non-unique.
```

Comment:
```suggestion
Fix uploading multiple cross signing signatures for the same user.
```

Or something. Broadly the changelogs are aimed at admins of servers rather than synapse devs, so should talk about the changes in terms of impact to them.